### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by st-schulte**
I did a German translation for the plugin. Perhaps there is interest to transfer this into the main project.

### CODE REVIEW

File: locale/de_DE/locale.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the de_DE (Deutsch) locale.
```
Comment (mtub) [inline locale/de_DE/locale.xml]: Deutsch -> Deutsch (Deutschland)

File: locale/de_DE/locale.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  -->
+
+<locale name="de_DE" full_name="Deutsch (Deutschland)">
+	<message key="plugins.generic.browseBySection.name">Nach Rubriken durchblättern</message>
+	<message key="plugins.generic.browseBySection.description">Erlaubt Besuchern die Zeitschrift nach Rubriken durchzublättern.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: Erlaubt Besucher/innen, die Zeitschrift …

File: locale/de_DE/locale.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+	<!-- Section Browsing -->
+	<message key="plugins.generic.browseBySection.emptySection">In dieser Rubrik wurde bislang kein Beitrag publiziert.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: … Beitrag veröffentlicht.

File: locale/de_DE/locale.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+	<!-- Section Management -->
+	<message key="plugins.generic.browseBySection.browsingLabel">Nach Rubriken durchblättern</message>
+	<message key="plugins.generic.browseBySection.enableBrowsing">Ermöglicht den Zugriff auf das Archiv mit allen publizierten Artikel einer Rubrik.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: allen veröffentlichten Artikeln

File: locale/de_DE/locale.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	<message key="plugins.generic.browseBySection.browseByPathLabel">URL-Pfad der Rubrik</message>
+	<message key="plugins.generic.browseBySection.browseByPathDescription">Passe den URL-Pfad an, über den Besucher auf das Archiv der Rubrik zugreifen können. Der Pfad darf nur Buchstaben, Zahlen und Bindestriche enthalten.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: Besucher/innen

File: locale/de_DE/locale.xml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+	<message key="plugins.generic.browseBySection.browseByPerPageLabel">Beiträge pro Seite</message>
+	<message key="plugins.generic.browseBySection.browseByPerPageDescription">Zahl der Beiträge, die auf einer Seite des Rubrik-Archivs dargestellt werden.</message>
+	<message key="plugins.generic.browseBySection.browseByDescriptionLabel">Beschreibung des Rubrik-Archivs</message>
+
+	<!-- Section Navigation Menus -->
+	<message key="plugins.generic.browseBySection.navMenuItem">Rubrik: {$name}</message>
+	<message key="plugins.generic.browseBySection.navMenuItem.description">Link zu dem Archiv der publizierten Beiträge in dieser Rubrik.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: veröffentlichten Beiträge

**Review by mtub — CHANGES_REQUESTED** [review]
Thanks, Stefan. I have made a few comments to have this translation in sync with the rest of OJS, mostly having gender-neutral terms and replacing "publiziert" with "veröffentlicht". Could you update your pull request? (If you don't have the time I can do an update later…)

Danke!

**Comment by st-schulte** [comment]
Thanks for your review, Marco. I have changed the translations accordingly.

File: locale/de_DE/locale.xml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the de_DE Deutsch (Deutschland) locale.
+  -->
+
+<locale name="de_DE" full_name="Deutsch (Deutschland)">
+	<message key="plugins.generic.browseBySection.name">Nach Rubriken durchblättern</message>
+	<message key="plugins.generic.browseBySection.description">Erlaubt Besucher/innen die Zeitschrift nach Rubriken durchzublättern.</message>
```
Comment (mtub) [inline locale/de_DE/locale.xml]: There's a comma missing… mind sending a quick update? -> "Erlaubt Besucher/innen, die Zeitschrift…"

**Comment by st-schulte** [comment]
Done!

**Comment by mtub** [comment]
Thanks, Stefan. @NateWr, please go ahead and merge :+1:

**Comment by NateWr** [comment]
:+1: Thanks to both of you!

**Pull request merged**